tools/roxvet/analyzers/dontcompareproto/analyzer.go

```diff
@@ -0,0 +1,124 @@
+package donotcompareproto
```

Comment:
It would be nice if the package name and the analyzer directory name would match (I prefer donotcompareproto).